src/components/organisms/WithInformationCard/WithInformationCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { InformationLeftColumn } from "components/organisms/InformationLeftColumn";
+import InformationCard from "components/molecules/InformationCard";
+
+import "./WithInformationCard.scss";
+
+interface PropsType {
+  venue: AnyVenue;
+  showInfo?: boolean;
+}
+
+export const WithInformationCard: React.FunctionComponent<PropsType> = ({
+  venue,
+  showInfo = false,
+}) => (
+  <>
+    {showInfo && (
+      <InformationLeftColumn iconNameOrPath={venue?.host?.icon || "info"}>
+        <InformationCard>
+          <p className="information-title">{venue.name}</p>
+          {venue.config?.landingPageConfig.subtitle && (
+            <p className="information-subtitle">
+              {venue.config?.landingPageConfig.subtitle}
+            </p>
```

Comment:
Would it be better to just add a new optional `subtitle` prop to `InformationCard` and pass this value into it?